index.html

```diff
@@ -982,7 +982,13 @@ <h3>Contexts</h3>
 the context.
           </dd>
         </dl>
-
+        <p class="note">
+Though this specification requires that a <code>@context</code> <a>property</a>
+be present, there is no requirement that the value of the <code>@context</code>
+property be processed. This is to support compatibility with certain JSON
```

Comment:
Note quite correct.

For JSON-LD aware processors, you have to process the `@context`.

For JSON processors, you have to make sure the order of the values in `@context` is what you expect them to be for your particular application (but you don't have to use the values to do JSON-LD based processing). So, JSON-based processors have to look for something like this:

```
  "@context": ["https://www.w3.org/2018/credentials/v1", EXPECTED_CONTEXT_2, EXPECTED_CONTEXT_3]
```

Reply:
I was (also) going to add some text to the Syntactic Sugar section of the spec to clarify this point (possibly with an example of what a JSON-based processor should do): https://w3c.github.io/vc-data-model/#syntactic-sugar

Reply:
I've made some changes in [b6088de](https://github.com/w3c/vc-data-model/pull/535/commits/b6088dea301b11deba1b3ddac4434074e691a4b3)
(I'm not feeling qualified to make changes to the syntactic sugar section, but would welcome a PR to my PR)

Reply:
Looks OK now